src/components/user/LoginDescription.scss

```diff
@@ -0,0 +1,56 @@
+@import '../../styles/styles.scss';
+
+
+.titleLogin {
+
+    margin-top: 3%;
+    margin-bottom: 5%;
+
+    font-family: Open Sans;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 20px;
```

Comment:
You can use this as a GUIDE: https://github.com/rootstrap/tech-guides/blob/master/css.md

You can break the rules on this guide, but it can help you to choose between different units.

Reply:
I was actually using the units and sizes from Figma's mock. Should I ignore them then?